partiql-cli/README.adoc

```diff
@@ -55,6 +59,46 @@ man partiql
 
     --strict
         Strict mode (type-checking) throws runtime exceptions on mistyped operations.
+
+**Loading PartiQL Environment files**
```

Comment:
thanks for including the example. let's include this under a separate heading/code block. it's currently within the `man partiql` description which can be confusing